CMake/exec2man.sh

```diff
@@ -68,13 +68,13 @@ synopsis="$name [-h] [-v] $reqoptions $options";
 ./"$name" -h | \
   sed 's/^For further information/Additional Information\n\n For further information/' | \
   sed 's/^consult the documentation/ consult the documentation/' | \
-  sed 's/^distribution of MLPACK./ distribution of MLPACK./' | \
+  sed 's/^distribution of mlpack./ distribution of mlpack./' | \
   awk -v syn="$synopsis" \
       '{ if (NR == 1) print "NAME\n '"$name"' - "tolower($0)"\nSYNOPSIS\n "syn" \nDESCRIPTION\n" ; else print } ' | \
   sed '/^[^ ]/ y/qwertyuiopasdfghjklzxcvbnm:/QWERTYUIOPASDFGHJKLZXCVBNM /' | \
   sed 's/  / /g' | \
-  awk '/NAME/,/REQUIRED OPTIONS/ { print; } /ADDITIONAL INFORMATION/,0 { print; } /REQUIRED OPTIONS/,/ADDITIONAL INFORMATION/ { if (!/REQUIRED_OPTIONS/ && !/OPTIONS/ && !/ADDITIONAL INFORMATION/) { if (/ --/) { printf "\n" } sub(/^[ ]*/, ""); sub(/ [ ]*/, " "); printf "%s ", $0; } else { if (!/REQUIRED OPTIONS/ && !/ADDITIONAL INFORMATION/) { print "\n"$0; } } }' | \
+  awk '/NAME/,/.*OPTIONS/ { if (!/.*OPTIONS/) { print; } } /ADDITIONAL INFORMATION/,0 { print; } /.*OPTIONS/,/ADDITIONAL INFORMATION/ { if (!/REQUIRED INPUT OPTIONS/ && !/OPTIONAL INPUT OPTIONS/ && !/OPTIONAL OUTPUT OPTIONS/ && !/ADDITIONAL INFORMATION/) { if (/ --/) { printf "\n" } sub(/^[ ]*/, ""); sub(/ [ ]*/, " "); printf "%s ", $0; } else { if (!/ADDITIONAL INFORMATION/) { print "\n"$0; } } }' | \
   sed 's/  ADDITIONAL INFORMATION/\n\nADDITIONAL INFORMATION/' | \
-  txt2man -P mlpack -t "$name" -d 1 | \
+  txt2man -t "$name" -s 1 -r "MLPACK Utilities" -v "User Commands" | \
```

Comment:
Ah, I didn't realize you had updated this PR.  Do you want to change the following bit for the release name and version?

```
-r "mlpack-$version"
```

and then somewhere before this call the bit to extract the version:

```
# Get the version.
version=`./"$name" --version | sed 's/^.* \([^ ]*\)\.$/\1/'`;
```

I can also easily do that during the merge process---up to you.